### DISCUSSION TITLE
Daily 05/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#541] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by rpgrca** [comment]
Ayer implementé algo del control de acceso via Django, sirve pero no hay forma de probarlo porque ahora tenemos control de acceso via Angular así que eso tiene prioridad. Sin embargo si pongo que la API de artículos es IsAdminOnly no retorna nada cuando estás como invitado o cliente así que eso estaría funcionando. Hoy voy a empezar de a poco con lo de ventas.

**Comment by BraianTroncoso** [comment]
Ayer me asigné un par de tareas para realizar durante la semana. Estuve con un par de cuestiones laborales, hoy también tengo trabajo, asi que en el transcurso del día voy a ir haciendo las issues pretendidas.
También tengo clases de Python.
